src/test/java/guitests/guihandles/PersonCardHandle.java

```
@@ -68,4 +68,13 @@ public String getEmail() {
                 .map(Label::getText)
                 .collect(Collectors.toList());
     }
+
+    public List<String> getTagStyleClasses(String tag) {
+        return tagLabels
+                .stream()
+                .filter(label -> label.getText().equals(tag))
+                .map(Label::getStyleClass)
+                .findFirst()
+                .orElse(null);
```

we should throw exception instead to make the test fail faster :)

---

What should I throw here? We don't have a `NoSuchTagException` class or something.

---

`IllegalArgumentException` should do. See [here](https://github.com/se-edu/addressbook-level4/blob/master/src/test/java/seedu/address/logic/parser/CommandParserTestUtil.java#L23).